Neos.FluidAdaptor/Classes/ViewHelpers/Format/JsonViewHelper.php

```diff
@@ -89,4 +91,26 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
 
         return json_encode($value, $options);
     }
+
+    /**
+     * @param string $argumentsName
+     * @param string $closureName
+     * @param string $initializationPhpCode
+     * @param ViewHelperNode $node
+     * @param TemplateCompiler $compiler
+     * @return string
+     */
+    public function compile($argumentsName, $closureName, &$initializationPhpCode, ViewHelperNode $node, TemplateCompiler $compiler)
+    {
+        $valueVariableName = $compiler->variableName('value');
+        $optionsVariableName = $compiler->variableName('options');
+        $initializationPhpCode .= sprintf('%1$s = (%2$s[\'value\'] !== null ? %2$s[\'value\'] : %3$s());', $valueVariableName, $argumentsName, $closureName) . chr(10);
+        $initializationPhpCode .= sprintf('%1$s = %2$s[\'forceObject\'] !== false ? JSON_HEX_TAG |JSON_FORCE_OBJECT : JSON_HEX_TAG;', $optionsVariableName, $argumentsName) . chr(10);
+
+        return sprintf(
+            'json_encode(%1$s, %2$s)',
+            $valueVariableName,
+            $optionsVariableName
+        );
+    }
```

Comment:
yeah that wasn't clear, before when the renderStatic method was still there it was basically the same code just reformatted and would be cool if it could do the reformatting itself.. but since it's replaced that doesn't make sense anymore